src/cmd/internal.go

```diff
@@ -37,8 +37,23 @@ var configSchemaCmd = &cobra.Command{
 	},
 }
 
+var configActionsSchemaCmd = &cobra.Command{
+	Use:     "config-actions-schema",
+	Aliases: []string{"c"},
+	Short:   lang.CmdInternalConfigSchemaShort,
+	Run: func(cmd *cobra.Command, args []string) {
+		schema := jsonschema.Reflect(&types.ActionsFile{})
```

Comment:
I opted to use comment descriptions instead of shoving more things into the struct tag, to get those comments in the schema you either need to use the same reflector as wren or move the comments back into the struct tags, I'd prefer to leave then as comments since it's much more readable

Reply:
I think it's def a stylistic thing, I actually prefer the struct tags because:
- it's one less reflective operation
- the comments look like comments, it's not clear that they serve another purpose without looking at another part of the codebase

Thoughts @decleaver ?

Reply:
Yeah, having comments that are more than just comments (aka actually matter) makes me nervous

Reply:
I avoid reflecting as much as possible.